examples/material_gallery/lib/time_picker_demo.dart

```diff
@@ -29,7 +29,7 @@ class _TimePickerDemoState extends State<TimePickerDemo> {
 
   Widget build(BuildContext context) {
     return new Column([
-      new Text('${_selectedTime.hour}:${_selectedTime.minute}'),
+      new Text('${_selectedTime}'),
```

Comment:
this should have given you an analyzer lint (no need for the braces in this interpolation)